tests/e2e/plugins/demo.php

```diff
@@ -0,0 +1,10 @@
+<?php
+/**
+ * Plugin Name: E2E Tests Demo Plugin
+ * Plugin URI:  https://github.com/ampproject/amp-wp
+ * Description: Demo Plugin that can be installed during E2E tests.
```

Comment:
> For example, such a plugin could create a custom post type and the accompanying E2E test would verify that block validation errors are shown for this custom post type too.

@swissspidy Could we add this clarifying example to the description here, too?

Reply:
@miina Sure. Maybe in `contributing.md` though?

Reply:
Yep, makes sense 👍

Reply:
Done in 2fdebe1ed017294154c015864e656c4a94b9f037